src/test/java/marquez/TestJobSerialization.java

```diff
@@ -0,0 +1,38 @@
+package marquez;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jackson.Jackson;
+import marquez.core.Job;
+import org.junit.Test;
+import static io.dropwizard.testing.FixtureHelpers.fixture;
+
+import java.sql.Timestamp;
+
+public class TestJobSerialization {
+    private static final ObjectMapper MAPPER = Jackson.newObjectMapper();
+
+    public Job createSimepleJob() {
```

Comment:
Maybe instead of a method, we can define a static variable `JOB`?

```java
static final Job JOB = new Job(
  "first-best-job",
  new Timestamp(1532038468530L),
  new Timestamp(1532038468530L),
  1,
  2,
  new Timestamp(1532036468530L),
  Boolean.TRUE,
 "Best job. First job.");
```
then, we can reference `JOB` in the tests. For example, in line `28`:

```java
Assert.assertEquals(MAPPER.writeValueAsString(JOB), expected);
```